src/spec_tools/transform.cljc

```diff
@@ -190,7 +205,7 @@
      :string keyword->string}
     #?(:clj
        {:uri string->uri
-        :bigdec string->decimal
+        :bigdec (number-or-string-> string->decimal)
```

Comment:
instead of converting the value first to string and then to decimal, it could be just coerced using `bigdec` => much faster. I'll do that after the merge.